### ISSUE TITLE
IE9: classList shim fails on SVG elements

### ISSUE THREAD

**Issue by shawnbot** [WebReflection/dom4#10]
SVG elements' `className` attribute is not a string, so [this line](https://github.com/WebReflection/dom4/blob/master/src/dom4.js#L182) fails. I've had success patching it to read:

```js
var className = (node.className.baseVal || node.className).replace(trim, '');
```

but there's probably a faster way to determine which one to use.

**Comment by WebReflection** [comment]
typeof might be a better option, will try to debug this soon, thanks

**Comment by ospi** [comment]
Hi
I experienced the same problem. Will this fix be included in the next release? When will the next version be released?

**Comment by WebReflection** [comment]
Apologies I've completely forgotten about this one, it's fixed with tests included in version `1.3.3`

It might take a little while before it propagates through the CDN
